src/frontends/tensorflow/src/utils.cpp

```diff
@@ -27,3 +28,20 @@ void ov::frontend::tensorflow::set_node_name(const std::string& node_name, const
 void ov::frontend::tensorflow::set_out_name(const std::string& out_name, const ov::Output<ov::Node>& output) {
     output.get_tensor().add_names({out_name});
 }
+
+ov::op::PadType convert_deconv_tf_padding(const ov::frontend::tensorflow::NodeContext& node, const std::string& tf_padding) {
```

Comment:
I guess this method can be useful not only for deconv op but for all convolutions to get  auto_pad(padding_type) attribute. Please rename this.

Reply:
Not at all, for Convolution operation SAME maps to SAME_UPPER. The different logic:)

Reply:
ok, but we can check type of the operation here and return relevant value.
It looks like the function for convolutions will be duplicate most logic written here

Reply:
Ok, let us do this

Reply:
Fixed. Thanks.